provider/block_dirty_bitmap.py

```diff
@@ -25,13 +25,17 @@ def parse_params(vm, params):
         target_image_params = params.object_params(target_image)
         json_backend_list = ["ceph", "iscsi_direct"]
         if target_image_params["image_backend"] in json_backend_list:
+        if "rbd_pool_name" in target_image_params:
             get_image_name = qemu_storage.get_image_json
             target_image_filename = get_image_name(target_image,
                                                    target_image_params,
                                                    data_dir.get_data_dir())
         else:
             target_image_filename = storage.get_image_filename(
                 target_image_params, data_dir.get_data_dir())
+            get_image_name = storage.get_image_filename
+            target_image_filename = get_image_name(target_image_params,
```

Comment:
dup with line 34-35

Reply:
Maybe after the conflict was resolved, the code was not pushed. Have updated it to latest. Thanks, zhenchao.

Reply:
Hi, zhenchao
   After the confilication(with patch 3224) has been resolved, no update of this file, so it has been removed from this patch, please take a double check.

BR,
Aliang